src/components/v2/Modal/Modal.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement } from 'react';
+import Backdrop from '@mui/material/Backdrop';
+import { Button, Modal as MUIModal, ModalProps } from '@mui/material';
+import Fade from '@mui/material/Fade';
+import Box from '@mui/material/Box';
+import { useModalStyles } from './ModalStyles';
+
+export interface IModalProps extends ModalProps {
+  children: any;
```

Comment:
You don't need to specify this prop, since you're extending `React.FC` it will already be included